src/main/kotlin/domain/ProfitCalculator.kt

```diff
@@ -0,0 +1,52 @@
+package domain
+
+class ProfitCalculator(
+    private val players: Players
+) {
+
+    fun getDealerProfit(): Double {
+        var dealerProfit = NONE_PROFIT
+        players.users.forEach { user ->
+            dealerProfit += singleUserDealerProfit(user)
+        }
+        return dealerProfit
```

Comment:
```suggestion
        return players.users.sumOf { user ->
            singleUserDealerProfit(user)
        }
```
sumOf를 활용하면 좀 더 간결한 코드를 작성할 수 있어요.